README.md

```diff
@@ -25,9 +25,9 @@ Experiment tracking for Keras-trained models.
 ## Resources
 
 * [Documentation](https://docs.neptune.ai/integrations/keras)
-* [Code example on GitHub](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/tensorflow-keras/scripts)
-* [Runs logged in the Neptune app](https://app.neptune.ai/o/common/org/tf-keras-integration/e/TFK-18/all)
-* [Run example in Google Colab](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/tensorflow-keras/notebooks/Neptune_TensorFlow_Keras.ipynb)
+* [Code example on GitHub](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/tensorflow-keras)
+* [Runs logged in the Neptune app](https://neptune.ai/resources/tensorflow-keras-integration-example)
```

Comment:
Why not keep the direct app link?

Reply:
I saw that this approach was used in the newer repositories (detectron2 for example). Makes it easier to maintain I guess as then only the marketing resource has to be updated in case of changes

Reply:
Huh, not sure I'd noted that. I don't think it's good to present the target as being in the Neptune app when it's not, though.

Wouldn't one option be to just link to the showcase project and make sure the pinned view shows appropriate example runs?

Reply:
No strong opinions here. @Patrycja-J - thoughts?

Reply:
I would be in favour of directly linking to the Custom dashboard in the showcase run, and not just to the project.